spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -11,6 +11,7 @@
 # TODO: add test to make sure that all postprocessing flags match the core.DEFAULT dictionary items
 # TODO: Fetch default value (and display) depending on the model that is used.
 # TODO: accommodate multiclass segmentation
+# Intentionally broken URL to test URL checker: https://abcdefg.org/hijklmnop
```

Comment:
Looks like everything is working OK, 30s delay and all:

![image](https://github.com/user-attachments/assets/5ecfc280-735b-427d-9b00-816219c9c4d6)